com.unity.multiplayer.mlapi/Prototyping/NetworkAnimator.cs

```diff
@@ -235,8 +235,8 @@ private void SendAllParamsAndState(int animStateHash, float animStateTime)
                 {
                     Send = new ClientRpcSendParams
                     {
-                        TargetClientIds = NetworkingManager.Singleton.ConnectedClientsList
-                            .Where(c => c.ClientId != NetworkingManager.Singleton.ServerClientId)
+                        TargetClientIds = MLAPI.NetworkManager.Singleton.ConnectedClientsList
```

Comment:
Did you mean to scope these with MLAPI?

Reply:
ugh, no. I will have another look and updates things like these shortly.